src/native/corehost/corehost.proj

```diff
@@ -81,6 +81,7 @@
       <BuildArgs Condition="'$(Ninja)' == 'true'">$(BuildArgs) -ninja</BuildArgs>
       <BuildArgs>$(BuildArgs) -runtimeflavor $(RuntimeFlavor)</BuildArgs>
       <BuildArgs Condition="'$(OfficialBuildId)' != ''">$(BuildArgs) /p:OfficialBuildId="$(OfficialBuildId)"</BuildArgs>
+      <BuildArgs Condition="'$(OutputRid)' != ''">$(BuildArgs) -distrorid $(OutputRid)</BuildArgs>
```

Comment:
nit: this condition can be dropped since `OutputRid` is always set in this context via global msbuild props:
https://github.com/dotnet/runtime/blob/4c927373d91279096f82a19a0ff6e91c37e570a3/Directory.Build.props#L200